yurt/yurt_core/deploy.py

```diff
@@ -71,44 +78,41 @@ def deploy():
         else:
             raise OSError('Not in a Yurt project folder')
 
-    print("Option\tInventory")
-    for index, filename in enumerate(os.listdir("./orchestration/inventory")):
-        if not(filename == "vagrant"):
-            ENVIRONMENT_OPTIONS[str(index)] = filename
-            print("{0}:\t{1}".format(index, filename))
-    num_selection = raw_input_wrapper("Which environment do you want to deploy (use Option number)?:\t")
-    selection = ENVIRONMENT_OPTIONS[num_selection]
+    if selection is None:
+        print("Option\tInventory")
+        for index, filename in enumerate(os.listdir("./orchestration/inventory")):
+            if not(filename == "vagrant"):
```

Comment:
I feel like it didn't work for me in Python at one point and so I've just avoided using it. I'll try it out.